gax-java/.kokoro/release/publish_javadoc11.sh

```diff
@@ -1,76 +0,0 @@
-#!/bin/bash
```

Comment:
Does removing of this file mean that we are going to use the `publish_javadoc11.sh` in the root `.kokoro` folder(which we don't have yet)? Also if that's the case, we want to skip publishing the java doc for `gapic-generator-java` submodule as it is not supposed to be used by public.

Reply:
Yes, we will use the one at root. I'll add one soon. And yes, we'll skip publishing the javadoc for `gapic-generator-java`.